utils/contentful-posts.ts

```diff
@@ -26,12 +26,13 @@ const transformContentfulPosts = (items: object[] = []): IBlogPost[] => {
   const posts = items.map((item: any): IBlogPost => {
     return {
       id: item.sys.id,
-      title: item.fields.title,
-      slug: item.fields.slug,
+      author: item.fields.author,
+      body: formatBody(item.fields.body) || '',
```

Comment:
When we use a default value like this, we should ask ourselves if it isn't better to let the field to be optional instead.